Documentation/ceph-pool-crd.md

```diff
@@ -80,6 +80,9 @@ spec:
   mirroring:
     enabled: true
     mode: image
+    snapshotSchedule:
```

Comment:
Just for clarification, but does this mean that rook will only support a single snapshot schedule per-pool?

Reply:
Hum yes? So Ceph can have multiple schedules for a given pool?

Reply:
Yes, indeed. Plus you can apply schedules to individual images. The longer term goal is to be able to define named schedules and then just apply them (by-name) globally, to the pool, and/or to individual images as well.

Reply:
Rook does not apply schedules to individual images since it does not manage images. The only thing we can do is at the pool level. I guess we can make multiple schedules configurable.

Reply:
Understood and agreed. It would be nice if Rook and the future CSI can have a similar approach (CRD) for defining schedules, though.

Reply:
Thanks, I'll add multiple schedules for this patch anyway.